airflow-core/src/airflow/ui/src/layouts/Details/PanelButtons.tsx

```diff
@@ -100,6 +102,16 @@ const getWidthBasedConfig = (width: number, enableResponsiveOptions: boolean) =>
   };
 };
 
+const getVersionDisplayOptions = (translate: (key: string) => string) =>
+  createListCollection({
+    items: [
+      { label: translate("dag:panel.versionDisplay.options.showAll"), value: "all" },
+      { label: translate("dag:panel.versionDisplay.options.showDagVersion"), value: "dag" },
+      { label: translate("dag:panel.versionDisplay.options.showBundleVersion"), value: "bundle" },
+      { label: translate("dag:panel.versionDisplay.options.hideAll"), value: "none" },
```

Comment:
Let's make the values an enum const we can import in many places so we avoid future spelling mistakes?